scalac-scoverage-plugin/src/main/scala/scoverage/plugin.scala

```diff
@@ -35,16 +34,23 @@ class ScoveragePlugin(val global: Global) extends Plugin {
         options.dataDir = opt.substring("dataDir:".length)
       } else if (opt.startsWith("extraAfterPhase:") || opt.startsWith("extraBeforePhase:")) {
         // skip here, these flags are processed elsewhere
-      } else {
+      } else if (opt.startsWith("useEnvironment:")) {
```

Comment:
`userEnvironment` should be renamed to something more meaningful related to classpath `--use-classpath-dir`.

Reply:
This option should also be clear that it'll override `dataDir`.